velox/functions/sparksql/Arithmetic.h

```diff
@@ -318,4 +318,62 @@ struct IsNanFunction {
     }
   }
 };
+
+FOLLY_ALWAYS_INLINE static int8_t fromHex(char c) {
+  if (c >= '0' && c <= '9') {
+    return c - '0';
+  }
+
+  if (c >= 'A' && c <= 'F') {
+    return 10 + c - 'A';
+  }
+
+  if (c >= 'a' && c <= 'f') {
+    return 10 + c - 'a';
+  }
+  return -1;
+}
+
+template <typename T>
+struct FromHexFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE bool call(
+      out_type<Varchar>& result,
+      const arg_type<Varbinary>& input) {
```

Comment:
The signature looks different with the documentation, could you confirm? Here the input is varbinary and the output is varchar.

Reply:
already fixed, thanks!